docs/services/general-info/added-services.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Addes Services per Release
+date: 2024-07-12T00:00:00+00:00
+weight: 20
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/services/general-info
+geekdocFilePath: added-services.md
+geekdocCollapseSection: true
+---
+
+Over time, services get added with new releases published. This page gives an overview which services exist respectively
+have been added per release.
+
+## Added Services
+
+The following table gives an overview about releases and their services.
+
+| 3.0                | 4.0                | 5.0                | Rolling            |
+| ------------------ | ------------------ | ------------------ | ------------------ |
+|                    |                    |                    | activitylog        |
+| antivirus          | antivirus          | antivirus          | antivirus          |
+| app-provider       | app-provider       | app-provider       | app-provider       |
+| app-registry       | app-registry       | app-registry       | app-registry       |
+| audit              | audit              | audit              | audit              |
+| auth-basic         | auth-basic         | auth-basic         | auth-basic         |
+| auth-bearer        | auth-bearer        | auth-bearer        | auth-bearer        |
+| auth-machine       | auth-machine       | auth-machine       | auth-machine       |
+| eventhistory       | eventhistory       | auth-service       | auth-service       |
```

Comment:
`eventhistory` needs to go to line 31

Reply:
I dont get it, it is in line 31.
introduced with v5